transforms/angle-brackets/transform.js

```diff
@@ -124,6 +124,20 @@ const IGNORE_MUSTACHE_STATEMENTS = [
 
   // ember-app-version
   'app-version',
+
+  // ember-font-awesome
+  'fa-icon',
+  'fa-list',
+  'fa-stack',
+
+  // ember-svg-jar
+  'svg-jar',
+
+  // ember-concurrency
+  'perform',
+
+  // ember-maybe-in-element
+  'maybe-in-element',
```

Comment:
@Turbo87 It's high time we should move this list to a separate file, I thought of doing this earlier, but couldn't find time to work on it.

Reply:
good idea. done!

Reply:
The next improvement I want to make is to use the telemetry helper addon to get rid of the need to list helpers.

Reply:
can you open an issue for that here? would like to discuss the implications before we sink time into that 😉